gio/gresources/gresources.go

```diff
@@ -0,0 +1,55 @@
+package gresources
+
+// #include <gio/gio.h>
+// #include <stdlib.h>
+import "C"
+import (
+	"errors"
+	"unsafe"
+)
+
+// GResourceLookupFlags is a representation of GTK's GResourceLookupFlags
+type LookupFlags int
```

Comment:
please name it `ResourceLookupFlags`

Reply:
Done - https://github.com/gotk3/gotk3/pull/565/commits/31fcf3e164a4e6869b2ea420c9c8328ce0709eea